.github/workflows/ci-build.yaml

```diff
@@ -139,17 +127,15 @@ jobs:
         run: |
           tar -xzf ../maven-repo.tgz -C ~
       - name: Integration Tests
+        env:
+          TEST_MODULES: ${{matrix.tests}}
+        shell: bash
         run: |
-          EXAMPLE_PROJECTS=""
-          for EXAMPLE in $(yq -M -N e ".${{ matrix.category }}" .github/test-categories.yaml | cut -f2 -d' '); do
-            if [ "${EXAMPLE}" == "null" ]; then
-              continue
-            fi
-
+          for MODULE in ${TEST_MODULES//,/ }; do
             ./mvnw ${MAVEN_ARGS} clean verify \
               -Dformatter.skip -Dimpsort.skip \
               -Pnative,docker \
-              -f "${EXAMPLE}/pom.xml"
+              -f "${MODULE}/pom.xml"
           done
```

Comment:
Will this run all tests of the given group even if some non-last test fails? And will the GH Actions job fail if any of the mvnw invocations fails?

Reply:
> Will this run all tests of the given group even if some non-last test fails? And will the GH Actions job fail if any of the mvnw invocations fails?

I believe it will 'fail fast'. I.e the job will terminate on the first `mvnw` failure. To my knowledge, that's how its always worked in this project.

Maybe we could try to be smart and report any failures at the end.

Reply:
Indeed, that would be nice to have.

Reply:
Latest commit produces a simple summary of any build failures.